Training/Easy/defibrillators.rb

```diff
@@ -39,18 +39,18 @@ def find_closest(defibrilators_hash)
     closest_name
   end
 
-  def init_defibrilators(defibrilator)
-    d_long = parse(defibrilator[4])
-    d_lat = parse(defibrilator[5])
+  def init_defibrillators(defibrillator)
+    d_long = parse(defibrillator[4])
+    d_lat = parse(defibrillator[5])
     x = calculate_x(d_long, d_lat)
     y = calculate_y(d_lat)
-    defibrilators[defibrilator[1]] = distance(x, y)
+    defibrillators[defibrillator[1]] = distance(x, y)
```

Comment:
Inside the `distance` function, you can use Ruby's "power" operator - so instead of
```
x * x
```
you can do
```
x ** 2
```